contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -3378,4 +3378,25 @@ AS SELECT
     , CAST(0 as sys.BIGINT) AS sequence_number
     , CAST(0 as sys.BIT) AS is_contained
 WHERE FALSE;
-GRANT SELECT ON sys.availability_groups TO PUBLIC;
\ No newline at end of file
+GRANT SELECT ON sys.availability_groups TO PUBLIC;
+
+CREATE OR REPLACE VIEW sys.sequences 
+AS SELECT 
+    so.*,
+    CAST(0 as sys.sql_variant) AS start_value
+    , CAST(0 as sys.sql_variant) AS increment
+    , CAST(0 as sys.sql_variant) AS minimum_value
+    , CAST(0 as sys.sql_variant) AS maximum_value
+    , CAST(0 as sys.BIT) AS is_cycling
+    , CAST(0 as sys.BIT) AS is_cached
+    , CAST(0 as INT) AS cache_size
+    , CAST(0 as sys.TINYINT) AS system_type_id
```

Comment:
Although T-SQL specs say this should be tinyint, we should still keep this as int. The reason being that in future when we implement this view, we would most certainly identify this as an OID and we map OIDs to int. For example, sys.types has a column system_type_id which indicates the type oid and is of type int.

Reply:
Changed from sys.TINYINT to INT as our expected oid could be larger than 255